msal/tests/Test.MSAL.NET.Unit/TelemetryTests.cs

```diff
@@ -313,6 +313,7 @@ public void PiiLoggingEnabledTrue_TenantAndUserIdHashedTest()
 
         [TestMethod]
         [TestCategory("PiiLoggingEnabled set to false, TenantId & UserId set to null values")]
+        [Ignore] // Disable until we investigate and fix this broken test
```

Comment:
Could you please raise a GitHub issue to express that the text fails, and add the GitHub issue number in the Ignore attribute. This way this is traceable in both direction. When the test or tested product code is fixed, we'll remove the IgnoreAttribute
`` 
[TestMethod]
[Ignore(GitHubIssueNumber)]
public void SomeTestWhichGotIgnoredWithTraceabilityToGitHubIssueToInvestiigateAndFix()
{
``

Reply:
We don't need this anymore. We needed to move the TelemetryTests to core, so this GitHub issue is no longer relevant